invenio_assets/__init__.py

```diff
@@ -30,20 +24,120 @@
     >>> assets = InvenioAssets(app)
 
 During initalization two Flask extensions :class:`flask_assets.Environment` and
-:class:`flask_collect.Collect` are instantiated and configured. Also bundles
-specified in the entry point group called ``invenio_assets.bundles`` are
-:ref:`automatically registered <entry-points-loading>`.
+:class:`flask_collect.Collect` are instantiated and configured.
+
+In order to build the assets, you can choose between using:
+
+* :ref:`Flask-WebpackExt <use-flask-webpackext>`;
+* :ref:`Flask-Assets <use-flask-assets>` (it will be deprecated soon.
```

Comment:
We should say that it is deprecated and soon it will be removed see [here](https://github.com/inveniosoftware/invenio-assets/pull/87/files#diff-418b7952f8e789183478f9f966916afdR46).